bears/c_languages/GNUIndentBear.py

```diff
@@ -23,18 +23,160 @@ class GNUIndentBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=80,
                          use_spaces: bool=True,
+                         blank_lines_after_declarations: bool=False,
+                         blank_lines_after_procedures: bool=False,
+                         blank_lines_after_commas: bool=False,
+                         braces_on_if_line: bool=False,
+                         braces_on_func_def_line: bool=False,
+                         cuddle_else: bool=False,
+                         while_and_brace_on_same_line: bool=False,
+                         case_indentation: int=0,
+                         space_before_semicolon_after_empty_loop: bool=True,
+                         delete_optional_blank_lines: bool=False,
```

Comment:
right :+1:

2016-06-16 20:44 GMT+01:00 Lasse Schuirmann <notifications@github.com>:

> In bears/c_languages/GNUIndentBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/519#discussion_r67411484>
> :
>
> > @@ -23,18 +23,160 @@ class GNUIndentBear:
> >      def create_arguments(filename, file, config_file,
> >                           max_line_length: int=80,
> >                           use_spaces: bool=True,
> > +                         blank_lines_after_declarations: bool=False,
> > +                         blank_lines_after_procedures: bool=False,
> > +                         blank_lines_after_commas: bool=False,
> > +                         braces_on_if_line: bool=False,
> > +                         braces_on_func_def_line: bool=False,
> > +                         cuddle_else: bool=False,
> > +                         while_and_brace_on_same_line: bool=False,
> > +                         case_indentation: int=0,
> > +                         space_before_semicolon_after_empty_loop: bool=True,
> > +                         delete_optional_blank_lines: bool=False,
>
> I'd make this a true
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/519/files/d88abd9e6371173007240bef6fb8a02fa123726a#r67411484>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xou9W7UNPdfQpZT6q1GJUK3eFPkcNks5qMae1gaJpZM4I02l0>
> .
>